2021/02/20 11:31:51 [notice] 2412#18056: using the "select" event method
2021/02/20 11:31:51 [notice] 2412#18056: openresty/1.13.6.2
2021/02/20 11:31:51 [notice] 2412#18056: start worker processes
2021/02/20 11:31:51 [notice] 2412#18056: start worker process 22084
2021/02/20 11:31:51 [notice] 22084#18388: openresty/1.13.6.2
2021/02/20 11:31:51 [notice] 22084#18388: create thread 17532
2021/02/20 11:31:51 [notice] 22084#18388: create thread 228
2021/02/20 11:31:51 [notice] 22084#18388: create thread 3488
2021/02/20 11:32:57 [warn] 22084#17532: *3 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 11:32:57 [error] 22084#17532: *3 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 11:33:57 [warn] 22084#17532: *3 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 11:33:57 [error] 22084#17532: *3 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:20:06 [notice] 16568#20420: using the "select" event method
2021/02/20 15:20:06 [notice] 16568#20420: openresty/1.13.6.2
2021/02/20 15:20:06 [notice] 16568#20420: start worker processes
2021/02/20 15:20:06 [notice] 16568#20420: start worker process 17044
2021/02/20 15:20:06 [notice] 17044#4476: openresty/1.13.6.2
2021/02/20 15:20:06 [notice] 17044#4476: create thread 4036
2021/02/20 15:20:06 [notice] 17044#4476: create thread 20340
2021/02/20 15:20:06 [notice] 17044#4476: create thread 12956
2021/02/20 15:21:29 [warn] 17044#4036: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:21:29 [error] 17044#4036: *1 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:22:29 [warn] 17044#4036: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:22:29 [error] 17044#4036: *1 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:22:50 [error] 17044#4036: *1 lua entry thread aborted: runtime error: ./lualib/testcode/testlua.lua:20: attempt to concatenate local 'name' (a nil value)
stack traceback:
coroutine 0:
	./lualib/testcode/testlua.lua: in function <./lualib/testcode/testlua.lua:1>, client: 192.168.50.121, server: localhost, request: "GET /luatest HTTP/1.1", host: "192.168.50.121:8070"
2021/02/20 15:23:50 [warn] 17044#4036: *5 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:23:50 [error] 17044#4036: *5 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:24:50 [warn] 17044#4036: *5 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:24:50 [error] 17044#4036: *5 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:31:50 [emerg] 20960#13880: directive "set" is not terminated by ";" in ./conf/nginx.conf:60
2021/02/20 15:33:28 [notice] 3668#14960: using the "select" event method
2021/02/20 15:33:28 [notice] 3668#14960: openresty/1.13.6.2
2021/02/20 15:33:28 [notice] 3668#14960: start worker processes
2021/02/20 15:33:28 [notice] 3668#14960: start worker process 20116
2021/02/20 15:33:28 [notice] 20116#14352: openresty/1.13.6.2
2021/02/20 15:33:28 [notice] 20116#14352: create thread 15692
2021/02/20 15:33:28 [notice] 20116#14352: create thread 2088
2021/02/20 15:33:28 [notice] 20116#14352: create thread 15892
2021/02/20 15:33:32 [error] 20116#15692: *2 lua entry thread aborted: runtime error: ...Evn/openresty-1.13.6.2-win64/lualib/testcode/testlua.lua:20: attempt to concatenate local 'name' (a nil value)
stack traceback:
coroutine 0:
	...Evn/openresty-1.13.6.2-win64/lualib/testcode/testlua.lua: in function <...Evn/openresty-1.13.6.2-win64/lualib/testcode/testlua.lua:1>, client: 192.168.50.121, server: localhost, request: "GET /luatest HTTP/1.1", host: "192.168.50.121:8070"
2021/02/20 15:34:32 [warn] 20116#15692: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:34:32 [error] 20116#15692: *1 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:35:32 [warn] 20116#15692: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:35:32 [error] 20116#15692: *1 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:37:13 [notice] 14804#6140: using the "select" event method
2021/02/20 15:37:13 [notice] 14804#6140: openresty/1.13.6.2
2021/02/20 15:37:13 [notice] 14804#6140: start worker processes
2021/02/20 15:37:13 [notice] 14804#6140: start worker process 14320
2021/02/20 15:37:13 [notice] 14320#12208: openresty/1.13.6.2
2021/02/20 15:37:13 [notice] 14320#12208: create thread 20656
2021/02/20 15:37:13 [notice] 14320#12208: create thread 21248
2021/02/20 15:37:13 [notice] 14320#12208: create thread 21244
2021/02/20 15:37:16 [error] 14320#20656: *1 lua entry thread aborted: runtime error: ...Evn/openresty-1.13.6.2-win64/lualib/testcode/testlua.lua:19: attempt to concatenate local 'name' (a nil value)
stack traceback:
coroutine 0:
	...Evn/openresty-1.13.6.2-win64/lualib/testcode/testlua.lua: in function <...Evn/openresty-1.13.6.2-win64/lualib/testcode/testlua.lua:1>, client: 192.168.50.121, server: localhost, request: "GET /luatest HTTP/1.1", host: "192.168.50.121:8070"
2021/02/20 15:38:08 [notice] 18572#3744: using the "select" event method
2021/02/20 15:38:08 [notice] 18572#3744: openresty/1.13.6.2
2021/02/20 15:38:08 [notice] 18572#3744: start worker processes
2021/02/20 15:38:08 [notice] 18572#3744: start worker process 22180
2021/02/20 15:38:08 [notice] 22180#19144: openresty/1.13.6.2
2021/02/20 15:38:08 [notice] 22180#19144: create thread 5036
2021/02/20 15:38:08 [notice] 22180#19144: create thread 18468
2021/02/20 15:38:08 [notice] 22180#19144: create thread 21144
2021/02/20 15:39:28 [warn] 22180#5036: *2 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:39:28 [error] 22180#5036: *2 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:40:28 [warn] 22180#5036: *2 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:40:28 [error] 22180#5036: *2 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:42:49 [notice] 9312#9884: using the "select" event method
2021/02/20 15:42:49 [notice] 9312#9884: using the "select" event method
2021/02/20 15:42:49 [notice] 9312#9884: openresty/1.13.6.2
2021/02/20 15:42:49 [notice] 9312#9884: openresty/1.13.6.2
2021/02/20 15:42:49 [info] 9312#9884: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 15:42:49 [notice] 9312#9884: start worker processes
2021/02/20 15:42:49 [notice] 9312#9884: start worker processes
2021/02/20 15:42:49 [notice] 9312#9884: start worker process 1936
2021/02/20 15:42:49 [notice] 9312#9884: start worker process 1936
2021/02/20 15:42:49 [notice] 1936#18476: openresty/1.13.6.2
2021/02/20 15:42:49 [notice] 1936#18476: openresty/1.13.6.2
2021/02/20 15:42:49 [info] 1936#18476: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 15:42:49 [notice] 1936#18476: create thread 10380
2021/02/20 15:42:49 [notice] 1936#18476: create thread 10380
2021/02/20 15:42:49 [notice] 1936#18476: create thread 19540
2021/02/20 15:42:49 [notice] 1936#18476: create thread 19540
2021/02/20 15:42:49 [notice] 1936#18476: create thread 14920
2021/02/20 15:42:49 [notice] 1936#18476: create thread 14920
2021/02/20 15:42:54 [debug] 1936#10380: *1 [lua] testlua.lua:19: hello:123
2021/02/20 15:43:54 [info] 1936#10380: *2 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/20 15:43:55 [warn] 1936#10380: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:43:55 [warn] 1936#10380: *1 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:43:55 [error] 1936#10380: *1 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:43:55 [error] 1936#10380: *1 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:44:13 [info] 1936#10380: *1 client prematurely closed connection, so upstream connection is closed too while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:45:13 [warn] 1936#10380: *5 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:45:13 [warn] 1936#10380: *5 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:45:13 [error] 1936#10380: *5 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:45:13 [error] 1936#10380: *5 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:45:22 [debug] 1936#10380: *8 [lua] testlua.lua:19: hello:123
2021/02/20 15:45:22 [info] 1936#10380: *5 client prematurely closed connection, so upstream connection is closed too while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:46:22 [warn] 1936#10380: *8 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:46:22 [warn] 1936#10380: *8 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:46:22 [error] 1936#10380: *8 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:46:22 [error] 1936#10380: *8 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:47:22 [warn] 1936#10380: *8 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:47:22 [warn] 1936#10380: *8 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:47:22 [error] 1936#10380: *8 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:47:22 [error] 1936#10380: *8 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/luatest"
2021/02/20 15:50:32 [debug] 1936#10380: *11 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 15:50:32 [debug] 1936#10380: *11 [lua] log.lua:50: End log_by_lua_file...
2021/02/20 15:50:32 [info] 1936#10380: *12 client prematurely closed connection, so upstream connection is closed too while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.jpg"
2021/02/20 15:51:32 [warn] 1936#10380: *11 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:51:32 [warn] 1936#10380: *11 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:51:32 [error] 1936#10380: *11 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:51:32 [error] 1936#10380: *11 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:52:32 [warn] 1936#10380: *11 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:52:32 [warn] 1936#10380: *11 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:52:32 [error] 1936#10380: *11 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:52:32 [error] 1936#10380: *11 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:53:17 [debug] 1936#10380: *11 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 15:53:17 [debug] 1936#10380: *11 [lua] log.lua:50: End log_by_lua_file...
2021/02/20 15:54:06 [debug] 1936#10380: *16 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 15:54:06 [debug] 1936#10380: *16 [lua] log.lua:50: End log_by_lua_file...
2021/02/20 15:54:17 [warn] 1936#10380: *11 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:54:17 [warn] 1936#10380: *11 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:54:17 [error] 1936#10380: *11 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:54:17 [error] 1936#10380: *11 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:55:17 [warn] 1936#10380: *11 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:55:17 [warn] 1936#10380: *11 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:55:17 [error] 1936#10380: *11 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:55:17 [error] 1936#10380: *11 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/1.pdf"
2021/02/20 15:59:17 [notice] 10684#20508: using the "select" event method
2021/02/20 15:59:17 [notice] 10684#20508: using the "select" event method
2021/02/20 15:59:17 [notice] 10684#20508: openresty/1.13.6.2
2021/02/20 15:59:17 [notice] 10684#20508: openresty/1.13.6.2
2021/02/20 15:59:17 [info] 10684#20508: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 15:59:17 [notice] 10684#20508: start worker processes
2021/02/20 15:59:17 [notice] 10684#20508: start worker processes
2021/02/20 15:59:17 [notice] 10684#20508: start worker process 20060
2021/02/20 15:59:17 [notice] 10684#20508: start worker process 20060
2021/02/20 15:59:17 [notice] 20060#16424: openresty/1.13.6.2
2021/02/20 15:59:17 [notice] 20060#16424: openresty/1.13.6.2
2021/02/20 15:59:17 [info] 20060#16424: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 15:59:17 [notice] 20060#16424: create thread 20656
2021/02/20 15:59:17 [notice] 20060#16424: create thread 20656
2021/02/20 15:59:17 [notice] 20060#16424: create thread 21896
2021/02/20 15:59:17 [notice] 20060#16424: create thread 21896
2021/02/20 15:59:17 [notice] 20060#16424: create thread 13052
2021/02/20 15:59:17 [notice] 20060#16424: create thread 13052
2021/02/20 15:59:21 [debug] 20060#20656: *1 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 15:59:21 [debug] 20060#20656: *1 [lua] log.lua:50: End log_by_lua_file...
2021/02/20 16:00:08 [debug] 20060#20656: *1 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 16:00:08 [debug] 20060#20656: *1 [lua] log.lua:18: Content-Disposition:attachment;filename=pac.pac
2021/02/20 16:00:08 [debug] 20060#20656: *1 [lua] log.lua:20: reverse_value:cap.cap=emanelif;tnemhcatta
2021/02/20 16:00:08 [debug] 20060#20656: *1 [lua] log.lua:22: Found = index:8
2021/02/20 16:00:08 [debug] 20060#20656: *1 [lua] log.lua:26: Encoded_filename:pac.pac
2021/02/20 16:00:08 [debug] 20060#20656: *1 [lua] log.lua:28: Filename:pac.pac
2021/02/20 16:00:08 [info] 20060#20656: *1 [lua] log.lua:33: Capture a download request:[http://192.168.50.121:8070/nginx/2/pac], post a request to create download log. while logging request, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/20 16:00:08 [error] 20060#20656: *1 failed to run log_by_lua*: ...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:46: attempt to index local 'list' (a nil value)
stack traceback:
	...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:46: in function <...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:1> while logging request, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/20 16:00:08 [error] 20060#20656: *1 failed to run log_by_lua*: ...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:46: attempt to index local 'list' (a nil value)
stack traceback:
	...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:46: in function <...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:1> while logging request, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/20 16:00:21 [info] 20060#20656: *2 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/20 16:02:33 [notice] 21748#19548: using the "select" event method
2021/02/20 16:02:33 [notice] 21748#19548: using the "select" event method
2021/02/20 16:02:33 [notice] 21748#19548: openresty/1.13.6.2
2021/02/20 16:02:33 [notice] 21748#19548: openresty/1.13.6.2
2021/02/20 16:02:33 [info] 21748#19548: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 16:02:33 [notice] 21748#19548: start worker processes
2021/02/20 16:02:33 [notice] 21748#19548: start worker processes
2021/02/20 16:02:33 [notice] 21748#19548: start worker process 6928
2021/02/20 16:02:33 [notice] 21748#19548: start worker process 6928
2021/02/20 16:02:34 [notice] 6928#14140: openresty/1.13.6.2
2021/02/20 16:02:34 [notice] 6928#14140: openresty/1.13.6.2
2021/02/20 16:02:34 [info] 6928#14140: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 16:02:34 [notice] 6928#14140: create thread 11976
2021/02/20 16:02:34 [notice] 6928#14140: create thread 11976
2021/02/20 16:02:34 [notice] 6928#14140: create thread 20096
2021/02/20 16:02:34 [notice] 6928#14140: create thread 20096
2021/02/20 16:02:34 [notice] 6928#14140: create thread 15240
2021/02/20 16:02:34 [notice] 6928#14140: create thread 15240
2021/02/20 16:02:38 [debug] 6928#11976: *1 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 16:02:38 [debug] 6928#11976: *1 [lua] log.lua:32: End log_by_lua_file...
2021/02/20 16:02:42 [debug] 6928#11976: *1 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 16:02:42 [debug] 6928#11976: *1 [lua] log.lua:18: Content-Disposition:attachment;filename=pac.pac
2021/02/20 16:02:42 [debug] 6928#11976: *1 [lua] log.lua:20: reverse_value:cap.cap=emanelif;tnemhcatta
2021/02/20 16:02:42 [debug] 6928#11976: *1 [lua] log.lua:22: Found = index:8
2021/02/20 16:02:42 [debug] 6928#11976: *1 [lua] log.lua:26: Encoded_filename:pac.pac
2021/02/20 16:02:42 [debug] 6928#11976: *1 [lua] log.lua:28: Filename:pac.pac
2021/02/20 16:02:42 [debug] 6928#11976: *1 [lua] log.lua:32: End log_by_lua_file...
2021/02/20 16:03:38 [info] 6928#11976: *2 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/20 16:10:49 [debug] 6928#11976: *6 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 16:10:49 [debug] 6928#11976: *6 [lua] log.lua:18: Content-Disposition:attachment;filename=pac.pac
2021/02/20 16:10:49 [debug] 6928#11976: *6 [lua] log.lua:20: reverse_value:cap.cap=emanelif;tnemhcatta
2021/02/20 16:10:49 [debug] 6928#11976: *6 [lua] log.lua:22: Found = index:8
2021/02/20 16:10:49 [debug] 6928#11976: *6 [lua] log.lua:26: Encoded_filename:pac.pac
2021/02/20 16:10:49 [debug] 6928#11976: *6 [lua] log.lua:28: Filename:pac.pac
2021/02/20 16:10:49 [debug] 6928#11976: *6 [lua] log.lua:32: End log_by_lua_file...
2021/02/20 16:11:49 [info] 6928#11976: *5 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/20 16:36:52 [notice] 11092#15372: using the "select" event method
2021/02/20 16:36:52 [notice] 11092#15372: using the "select" event method
2021/02/20 16:36:52 [notice] 11092#15372: openresty/1.13.6.2
2021/02/20 16:36:52 [notice] 11092#15372: openresty/1.13.6.2
2021/02/20 16:36:52 [info] 11092#15372: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 16:36:52 [notice] 11092#15372: start worker processes
2021/02/20 16:36:52 [notice] 11092#15372: start worker processes
2021/02/20 16:36:52 [notice] 11092#15372: start worker process 19508
2021/02/20 16:36:52 [notice] 11092#15372: start worker process 19508
2021/02/20 16:36:52 [notice] 19508#18220: openresty/1.13.6.2
2021/02/20 16:36:52 [notice] 19508#18220: openresty/1.13.6.2
2021/02/20 16:36:52 [info] 19508#18220: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 16:36:52 [notice] 19508#18220: create thread 6208
2021/02/20 16:36:52 [notice] 19508#18220: create thread 6208
2021/02/20 16:36:52 [notice] 19508#18220: create thread 10132
2021/02/20 16:36:52 [notice] 19508#18220: create thread 10132
2021/02/20 16:36:52 [notice] 19508#18220: create thread 20524
2021/02/20 16:36:52 [notice] 19508#18220: create thread 20524
2021/02/20 16:37:07 [debug] 19508#6208: *1 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 16:37:07 [debug] 19508#6208: *1 [lua] log.lua:18: Content-Disposition:attachment;filename=pac.pac
2021/02/20 16:37:07 [debug] 19508#6208: *1 [lua] log.lua:20: reverse_value:cap.cap=emanelif;tnemhcatta
2021/02/20 16:37:07 [debug] 19508#6208: *1 [lua] log.lua:22: Found = index:8
2021/02/20 16:37:07 [debug] 19508#6208: *1 [lua] log.lua:25: length27
2021/02/20 16:37:07 [debug] 19508#6208: *1 [lua] log.lua:27: Encoded_filename:pac.pac
2021/02/20 16:37:07 [debug] 19508#6208: *1 [lua] log.lua:29: Filename:pac.pac
2021/02/20 16:37:07 [debug] 19508#6208: *1 [lua] log.lua:33: End log_by_lua_file...
2021/02/20 16:38:06 [info] 19508#6208: *2 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/20 17:03:40 [notice] 1148#21036: using the "select" event method
2021/02/20 17:03:40 [notice] 1148#21036: using the "select" event method
2021/02/20 17:03:40 [notice] 1148#21036: openresty/1.13.6.2
2021/02/20 17:03:40 [notice] 1148#21036: openresty/1.13.6.2
2021/02/20 17:03:40 [info] 1148#21036: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:03:40 [notice] 1148#21036: start worker processes
2021/02/20 17:03:40 [notice] 1148#21036: start worker processes
2021/02/20 17:03:40 [notice] 1148#21036: start worker process 19596
2021/02/20 17:03:40 [notice] 1148#21036: start worker process 19596
2021/02/20 17:03:41 [notice] 19596#20152: openresty/1.13.6.2
2021/02/20 17:03:41 [notice] 19596#20152: openresty/1.13.6.2
2021/02/20 17:03:41 [info] 19596#20152: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:03:41 [notice] 19596#20152: create thread 17464
2021/02/20 17:03:41 [notice] 19596#20152: create thread 17464
2021/02/20 17:03:41 [notice] 19596#20152: create thread 5092
2021/02/20 17:03:41 [notice] 19596#20152: create thread 5092
2021/02/20 17:03:41 [notice] 19596#20152: create thread 8276
2021/02/20 17:03:41 [notice] 19596#20152: create thread 8276
2021/02/20 17:04:17 [debug] 19596#17464: *1 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 17:04:17 [debug] 19596#17464: *1 [lua] log.lua:18: Content-Disposition:attachment;filename=pac.pac
2021/02/20 17:04:17 [debug] 19596#17464: *1 [lua] log.lua:20: reverse_value:cap.cap=emanelif;tnemhcatta
2021/02/20 17:04:17 [debug] 19596#17464: *1 [lua] log.lua:22: Found = index:8
2021/02/20 17:04:17 [debug] 19596#17464: *1 [lua] log.lua:25: length27
2021/02/20 17:04:17 [debug] 19596#17464: *1 [lua] log.lua:29: nil
2021/02/20 17:04:17 [debug] 19596#17464: *1 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/20 17:04:17 [debug] 19596#17464: *1 [lua] log.lua:33: Filename:pac.pac
2021/02/20 17:04:17 [debug] 19596#17464: *1 [lua] log.lua:37: End log_by_lua_file...
2021/02/20 17:05:17 [info] 19596#17464: *2 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/20 17:10:59 [notice] 20808#21108: using the "select" event method
2021/02/20 17:10:59 [notice] 20808#21108: using the "select" event method
2021/02/20 17:10:59 [notice] 20808#21108: openresty/1.13.6.2
2021/02/20 17:10:59 [notice] 20808#21108: openresty/1.13.6.2
2021/02/20 17:10:59 [info] 20808#21108: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:10:59 [notice] 20808#21108: start worker processes
2021/02/20 17:10:59 [notice] 20808#21108: start worker processes
2021/02/20 17:10:59 [notice] 20808#21108: start worker process 18368
2021/02/20 17:10:59 [notice] 20808#21108: start worker process 18368
2021/02/20 17:10:59 [notice] 18368#1064: openresty/1.13.6.2
2021/02/20 17:10:59 [notice] 18368#1064: openresty/1.13.6.2
2021/02/20 17:10:59 [info] 18368#1064: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:10:59 [notice] 18368#1064: create thread 19936
2021/02/20 17:10:59 [notice] 18368#1064: create thread 19936
2021/02/20 17:10:59 [notice] 18368#1064: create thread 20632
2021/02/20 17:10:59 [notice] 18368#1064: create thread 20632
2021/02/20 17:10:59 [notice] 18368#1064: create thread 12012
2021/02/20 17:10:59 [notice] 18368#1064: create thread 12012
2021/02/20 17:11:07 [debug] 18368#19936: *2 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 17:11:07 [debug] 18368#19936: *2 [lua] log.lua:18: Content-Disposition:attachment;filename=pac.pac
2021/02/20 17:11:07 [debug] 18368#19936: *2 [lua] log.lua:20: reverse_value:cap.cap=emanelif;tnemhcatta
2021/02/20 17:11:07 [debug] 18368#19936: *2 [lua] log.lua:22: Found = index:8
2021/02/20 17:11:07 [debug] 18368#19936: *2 [lua] log.lua:25: length27
2021/02/20 17:11:07 [debug] 18368#19936: *2 [lua] log.lua:35: Encoded_filename:pac.pac
2021/02/20 17:11:07 [debug] 18368#19936: *2 [lua] log.lua:38: Filename:pac.pac
2021/02/20 17:11:07 [debug] 18368#19936: *2 [lua] log.lua:42: End log_by_lua_file...
2021/02/20 17:12:07 [info] 18368#19936: *1 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/20 17:12:45 [debug] 18368#19936: *4 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 17:12:45 [debug] 18368#19936: *4 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac"
2021/02/20 17:12:45 [debug] 18368#19936: *4 [lua] log.lua:20: reverse_value:"cap.cap"=emanelif;tnemhcatta
2021/02/20 17:12:45 [debug] 18368#19936: *4 [lua] log.lua:22: Found = index:10
2021/02/20 17:12:45 [debug] 18368#19936: *4 [lua] log.lua:25: length29
2021/02/20 17:12:45 [debug] 18368#19936: *4 [lua] log.lua:35: Encoded_filename:
2021/02/20 17:12:45 [debug] 18368#19936: *4 [lua] log.lua:38: Filename:
2021/02/20 17:12:45 [debug] 18368#19936: *4 [lua] log.lua:42: End log_by_lua_file...
2021/02/20 17:13:44 [info] 18368#19936: *5 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/20 17:14:25 [notice] 22304#13712: using the "select" event method
2021/02/20 17:14:25 [notice] 22304#13712: using the "select" event method
2021/02/20 17:14:25 [notice] 22304#13712: openresty/1.13.6.2
2021/02/20 17:14:25 [notice] 22304#13712: openresty/1.13.6.2
2021/02/20 17:14:25 [info] 22304#13712: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:14:25 [notice] 22304#13712: start worker processes
2021/02/20 17:14:25 [notice] 22304#13712: start worker processes
2021/02/20 17:14:25 [notice] 22304#13712: start worker process 11032
2021/02/20 17:14:25 [notice] 22304#13712: start worker process 11032
2021/02/20 17:14:25 [notice] 11032#5496: openresty/1.13.6.2
2021/02/20 17:14:25 [notice] 11032#5496: openresty/1.13.6.2
2021/02/20 17:14:25 [info] 11032#5496: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:14:25 [notice] 11032#5496: create thread 20524
2021/02/20 17:14:25 [notice] 11032#5496: create thread 20524
2021/02/20 17:14:25 [notice] 11032#5496: create thread 20272
2021/02/20 17:14:25 [notice] 11032#5496: create thread 20272
2021/02/20 17:14:25 [notice] 11032#5496: create thread 16200
2021/02/20 17:14:25 [notice] 11032#5496: create thread 16200
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac"
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:20: reverse_value:"cap.cap"=emanelif;tnemhcatta
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:22: Found = index:10
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:25: length29
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:32: 1111"pac.pac"
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:34: 11111
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:32: 1111pac.pac"
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:34: 11118
2021/02/20 17:14:33 [debug] 11032#20524: *2 [lua] log.lua:32: 1111
2021/02/20 17:14:33 [error] 11032#20524: *2 failed to run log_by_lua*: ...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:34: attempt to concatenate local 'start' (a nil value)
stack traceback:
	...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:34: in function <...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:1> while logging request, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/20 17:14:33 [error] 11032#20524: *2 failed to run log_by_lua*: ...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:34: attempt to concatenate local 'start' (a nil value)
stack traceback:
	...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:34: in function <...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:1> while logging request, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/20 17:15:33 [info] 11032#20524: *1 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/20 17:17:23 [notice] 21348#6316: using the "select" event method
2021/02/20 17:17:23 [notice] 21348#6316: using the "select" event method
2021/02/20 17:17:23 [notice] 21348#6316: openresty/1.13.6.2
2021/02/20 17:17:23 [notice] 21348#6316: openresty/1.13.6.2
2021/02/20 17:17:23 [info] 21348#6316: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:17:23 [notice] 21348#6316: start worker processes
2021/02/20 17:17:23 [notice] 21348#6316: start worker processes
2021/02/20 17:17:23 [notice] 21348#6316: start worker process 11124
2021/02/20 17:17:23 [notice] 21348#6316: start worker process 11124
2021/02/20 17:17:23 [notice] 11124#20232: openresty/1.13.6.2
2021/02/20 17:17:23 [notice] 11124#20232: openresty/1.13.6.2
2021/02/20 17:17:23 [info] 11124#20232: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:17:23 [notice] 11124#20232: create thread 20364
2021/02/20 17:17:23 [notice] 11124#20232: create thread 20364
2021/02/20 17:17:23 [notice] 11124#20232: create thread 15708
2021/02/20 17:17:23 [notice] 11124#20232: create thread 15708
2021/02/20 17:17:23 [notice] 11124#20232: create thread 20960
2021/02/20 17:17:23 [notice] 11124#20232: create thread 20960
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac"
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:20: reverse_value:"cap.cap"=emanelif;tnemhcatta
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:22: Found = index:10
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:25: length29
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:32: 1111"pac.pac"
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:34: 11111
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:32: 1111pac.pac"
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:34: 11118
2021/02/20 17:17:33 [debug] 11032#20524: *5 [lua] log.lua:32: 1111
2021/02/20 17:17:33 [error] 11032#20524: *5 failed to run log_by_lua*: ...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:34: attempt to concatenate local 'start' (a nil value)
stack traceback:
	...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:34: in function <...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:1> while logging request, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/20 17:17:33 [error] 11032#20524: *5 failed to run log_by_lua*: ...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:34: attempt to concatenate local 'start' (a nil value)
stack traceback:
	...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:34: in function <...hat/Evn/openresty-1.13.6.2-win64/lualib/testcode/log.lua:1> while logging request, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"


2021/02/20 17:18:01 [notice] 19904#20500: using the "select" event method
2021/02/20 17:18:01 [notice] 19904#20500: using the "select" event method
2021/02/20 17:18:01 [notice] 19904#20500: openresty/1.13.6.2
2021/02/20 17:18:01 [notice] 19904#20500: openresty/1.13.6.2
2021/02/20 17:18:01 [info] 19904#20500: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:18:01 [notice] 19904#20500: start worker processes
2021/02/20 17:18:01 [notice] 19904#20500: start worker processes
2021/02/20 17:18:01 [notice] 19904#20500: start worker process 9452
2021/02/20 17:18:01 [notice] 19904#20500: start worker process 9452
2021/02/20 17:18:01 [notice] 9452#20512: openresty/1.13.6.2
2021/02/20 17:18:01 [notice] 9452#20512: openresty/1.13.6.2
2021/02/20 17:18:01 [info] 9452#20512: OS: 300000 build:18363, "", suite:300, type:1
2021/02/20 17:18:01 [notice] 9452#20512: create thread 5424
2021/02/20 17:18:01 [notice] 9452#20512: create thread 5424
2021/02/20 17:18:01 [notice] 9452#20512: create thread 22408
2021/02/20 17:18:01 [notice] 9452#20512: create thread 22408
2021/02/20 17:18:01 [notice] 9452#20512: create thread 12824
2021/02/20 17:18:01 [notice] 9452#20512: create thread 12824
2021/02/20 17:18:07 [debug] 9452#5424: *2 [lua] log.lua:3: In log_by_lua_file...
2021/02/20 17:18:07 [debug] 9452#5424: *2 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac"
2021/02/20 17:18:07 [debug] 9452#5424: *2 [lua] log.lua:20: reverse_value:"cap.cap"=emanelif;tnemhcatta
2021/02/20 17:18:07 [debug] 9452#5424: *2 [lua] log.lua:22: Found = index:10
2021/02/20 17:18:07 [debug] 9452#5424: *2 [lua] log.lua:25: length29
2021/02/20 17:18:07 [debug] 9452#5424: *2 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/20 17:18:07 [debug] 9452#5424: *2 [lua] log.lua:34: Filename:pac.pac
2021/02/20 17:18:07 [debug] 9452#5424: *2 [lua] log.lua:38: End log_by_lua_file...
2021/02/20 17:19:07 [info] 9452#5424: *1 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/22 09:20:47 [notice] 15456#11052: using the "select" event method
2021/02/22 09:20:47 [notice] 15456#11052: using the "select" event method
2021/02/22 09:20:47 [notice] 15456#11052: openresty/1.13.6.2
2021/02/22 09:20:47 [notice] 15456#11052: openresty/1.13.6.2
2021/02/22 09:20:47 [info] 15456#11052: OS: 300000 build:18363, "", suite:300, type:1
2021/02/22 09:20:47 [notice] 15456#11052: start worker processes
2021/02/22 09:20:47 [notice] 15456#11052: start worker processes
2021/02/22 09:20:47 [notice] 15456#11052: start worker process 10296
2021/02/22 09:20:47 [notice] 15456#11052: start worker process 10296
2021/02/22 09:20:47 [notice] 10296#11888: openresty/1.13.6.2
2021/02/22 09:20:47 [notice] 10296#11888: openresty/1.13.6.2
2021/02/22 09:20:47 [info] 10296#11888: OS: 300000 build:18363, "", suite:300, type:1
2021/02/22 09:20:47 [notice] 10296#11888: create thread 15732
2021/02/22 09:20:47 [notice] 10296#11888: create thread 15732
2021/02/22 09:20:47 [notice] 10296#11888: create thread 18252
2021/02/22 09:20:47 [notice] 10296#11888: create thread 18252
2021/02/22 09:20:47 [notice] 10296#11888: create thread 7880
2021/02/22 09:20:47 [notice] 10296#11888: create thread 7880
2021/02/22 09:22:20 [debug] 10296#15732: *2 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:22:20 [debug] 10296#15732: *2 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:23:20 [info] 10296#15732: *1 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/22 09:27:51 [debug] 10296#15732: *3 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:27:51 [debug] 10296#15732: *3 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:27:51 [debug] 10296#15732: *3 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:27:51 [debug] 10296#15732: *3 [lua] log.lua:22: Found = index:11
2021/02/22 09:27:51 [debug] 10296#15732: *3 [lua] log.lua:25: length30
2021/02/22 09:27:51 [debug] 10296#15732: *3 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:27:51 [debug] 10296#15732: *3 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:27:51 [debug] 10296#15732: *3 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:28:50 [info] 10296#15732: *4 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/22 09:29:25 [debug] 10296#15732: *6 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *6 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:25 [debug] 10296#15732: *6 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:25 [debug] 10296#15732: *6 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:25 [debug] 10296#15732: *6 [lua] log.lua:25: length30
2021/02/22 09:29:25 [debug] 10296#15732: *6 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *6 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *6 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *8 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *8 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:25 [debug] 10296#15732: *8 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:25 [debug] 10296#15732: *8 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:25 [debug] 10296#15732: *8 [lua] log.lua:25: length30
2021/02/22 09:29:25 [debug] 10296#15732: *8 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *8 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *8 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *11 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *11 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:25 [debug] 10296#15732: *11 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:25 [debug] 10296#15732: *11 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:25 [debug] 10296#15732: *11 [lua] log.lua:25: length30
2021/02/22 09:29:25 [debug] 10296#15732: *11 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *11 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *11 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *18 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *18 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:25 [debug] 10296#15732: *18 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:25 [debug] 10296#15732: *18 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:25 [debug] 10296#15732: *18 [lua] log.lua:25: length30
2021/02/22 09:29:25 [debug] 10296#15732: *18 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *18 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *18 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *14 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *14 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:25 [debug] 10296#15732: *14 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:25 [debug] 10296#15732: *14 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:25 [debug] 10296#15732: *14 [lua] log.lua:25: length30
2021/02/22 09:29:25 [debug] 10296#15732: *14 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *14 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *14 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *15 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *15 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:25 [debug] 10296#15732: *15 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:25 [debug] 10296#15732: *15 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:25 [debug] 10296#15732: *15 [lua] log.lua:25: length30
2021/02/22 09:29:25 [debug] 10296#15732: *15 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *15 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *15 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *10 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *10 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:25 [debug] 10296#15732: *10 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:25 [debug] 10296#15732: *10 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:25 [debug] 10296#15732: *10 [lua] log.lua:25: length30
2021/02/22 09:29:25 [debug] 10296#15732: *10 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *10 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *10 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *20 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *20 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:25 [debug] 10296#15732: *20 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:25 [debug] 10296#15732: *20 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:25 [debug] 10296#15732: *20 [lua] log.lua:25: length30
2021/02/22 09:29:25 [debug] 10296#15732: *20 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *20 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *20 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *22 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:25 [debug] 10296#15732: *22 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:25 [debug] 10296#15732: *22 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:25 [debug] 10296#15732: *22 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:25 [debug] 10296#15732: *22 [lua] log.lua:25: length30
2021/02/22 09:29:25 [debug] 10296#15732: *22 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *22 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:25 [debug] 10296#15732: *22 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:26 [debug] 10296#15732: *22 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:26 [debug] 10296#15732: *22 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:26 [debug] 10296#15732: *22 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:26 [debug] 10296#15732: *22 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:26 [debug] 10296#15732: *22 [lua] log.lua:25: length30
2021/02/22 09:29:26 [debug] 10296#15732: *22 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:26 [debug] 10296#15732: *22 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:26 [debug] 10296#15732: *22 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:29 [debug] 10296#15732: *22 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:29 [debug] 10296#15732: *22 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:29 [debug] 10296#15732: *22 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:29 [debug] 10296#15732: *22 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:29 [debug] 10296#15732: *22 [lua] log.lua:25: length30
2021/02/22 09:29:29 [debug] 10296#15732: *22 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:29 [debug] 10296#15732: *22 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:29 [debug] 10296#15732: *22 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:36 [debug] 10296#15732: *26 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:36 [debug] 10296#15732: *26 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:36 [debug] 10296#15732: *26 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:36 [debug] 10296#15732: *26 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:36 [debug] 10296#15732: *26 [lua] log.lua:25: length30
2021/02/22 09:29:36 [debug] 10296#15732: *26 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:36 [debug] 10296#15732: *26 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:36 [debug] 10296#15732: *26 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:38 [debug] 10296#15732: *28 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:38 [debug] 10296#15732: *28 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:38 [debug] 10296#15732: *28 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:38 [debug] 10296#15732: *28 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:38 [debug] 10296#15732: *28 [lua] log.lua:25: length30
2021/02/22 09:29:38 [debug] 10296#15732: *28 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:38 [debug] 10296#15732: *28 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:38 [debug] 10296#15732: *28 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:29:43 [info] 10296#15732: *26 client 192.168.50.121 closed keepalive connection
2021/02/22 09:29:43 [info] 10296#15732: *28 client 192.168.50.121 closed keepalive connection
2021/02/22 09:29:44 [debug] 10296#15732: *22 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:29:44 [debug] 10296#15732: *22 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:29:44 [debug] 10296#15732: *22 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:29:44 [debug] 10296#15732: *22 [lua] log.lua:22: Found = index:11
2021/02/22 09:29:44 [debug] 10296#15732: *22 [lua] log.lua:25: length30
2021/02/22 09:29:44 [debug] 10296#15732: *22 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:29:44 [debug] 10296#15732: *22 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:29:44 [debug] 10296#15732: *22 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:30:44 [info] 10296#15732: *22 client 192.168.50.121 closed keepalive connection
2021/02/22 09:30:44 [debug] 10296#15732: *31 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:30:44 [debug] 10296#15732: *31 [lua] log.lua:18: Content-Disposition:attachment;filename="pac.pac;"
2021/02/22 09:30:44 [debug] 10296#15732: *31 [lua] log.lua:20: reverse_value:";cap.cap"=emanelif;tnemhcatta
2021/02/22 09:30:44 [debug] 10296#15732: *31 [lua] log.lua:22: Found = index:11
2021/02/22 09:30:44 [debug] 10296#15732: *31 [lua] log.lua:25: length30
2021/02/22 09:30:44 [debug] 10296#15732: *31 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 09:30:44 [debug] 10296#15732: *31 [lua] log.lua:34: Filename:pac.pac
2021/02/22 09:30:44 [debug] 10296#15732: *31 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:31:44 [info] 10296#15732: *31 client 192.168.50.121 closed keepalive connection
2021/02/22 09:32:37 [warn] 10296#15732: *33 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:32:37 [warn] 10296#15732: *33 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:32:37 [error] 10296#15732: *33 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:32:37 [error] 10296#15732: *33 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:32:41 [debug] 10296#15732: *36 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *36 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *37 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *37 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *38 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *38 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *39 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *39 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *40 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *40 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *41 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *41 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *42 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *42 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *43 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *43 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *44 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:32:41 [debug] 10296#15732: *44 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:33:13 [info] 10296#15732: *44 client prematurely closed connection, so upstream connection is closed too (10054: An existing connection was forcibly closed by the remote host) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:33:13 [debug] 10296#15732: *44 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:33:13 [debug] 10296#15732: *44 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:33:37 [warn] 10296#15732: *33 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:33:37 [warn] 10296#15732: *33 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:33:37 [error] 10296#15732: *33 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:33:37 [error] 10296#15732: *33 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:33:41 [info] 10296#15732: *33 client 192.168.50.121 closed keepalive connection
2021/02/22 09:33:49 [info] 10296#15732: *46 client prematurely closed connection, so upstream connection is closed too (10054: An existing connection was forcibly closed by the remote host) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:33:49 [debug] 10296#15732: *46 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:33:49 [debug] 10296#15732: *46 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:34:37 [info] 10296#15732: *50 client prematurely closed connection, so upstream connection is closed too (10054: An existing connection was forcibly closed by the remote host) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:34:37 [debug] 10296#15732: *50 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:34:37 [debug] 10296#15732: *50 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:34:43 [warn] 10296#15732: *48 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:34:43 [warn] 10296#15732: *48 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:34:43 [error] 10296#15732: *48 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:34:43 [error] 10296#15732: *48 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:34:54 [debug] 10296#15732: *53 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:34:54 [debug] 10296#15732: *53 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:35:25 [info] 10296#15732: *54 client prematurely closed connection, so upstream connection is closed too (10054: An existing connection was forcibly closed by the remote host) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:35:25 [debug] 10296#15732: *54 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:35:25 [debug] 10296#15732: *54 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:35:43 [warn] 10296#15732: *48 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:35:43 [warn] 10296#15732: *48 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:35:43 [error] 10296#15732: *48 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:35:43 [error] 10296#15732: *48 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:35:43 [info] 10296#15732: *48 client 192.168.50.121 closed keepalive connection
2021/02/22 09:36:01 [info] 10296#15732: *53 client prematurely closed connection, so upstream connection is closed too (10054: An existing connection was forcibly closed by the remote host) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:36:01 [debug] 10296#15732: *53 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:36:01 [debug] 10296#15732: *53 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:36:49 [info] 10296#15732: *57 client prematurely closed connection, so upstream connection is closed too (10054: An existing connection was forcibly closed by the remote host) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:36:49 [debug] 10296#15732: *57 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:36:49 [debug] 10296#15732: *57 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:37:38 [warn] 10296#15732: *59 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:37:38 [warn] 10296#15732: *59 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:37:38 [error] 10296#15732: *59 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:37:38 [error] 10296#15732: *59 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:38:09 [error] 10296#15732: *65 CreateFile() "D:/chrom/pac.pachttp:/192.168.50.83:10800/secure/Dashboard.jspa" failed (2: The system cannot find the file specified), client: 192.168.50.121, server: localhost, request: "GET /chrom/pac.pachttp://192.168.50.83:10800/secure/Dashboard.jspa HTTP/1.1", host: "192.168.50.121:8070"
2021/02/22 09:38:09 [error] 10296#15732: *65 CreateFile() "D:/chrom/pac.pachttp:/192.168.50.83:10800/secure/Dashboard.jspa" failed (2: The system cannot find the file specified), client: 192.168.50.121, server: localhost, request: "GET /chrom/pac.pachttp://192.168.50.83:10800/secure/Dashboard.jspa HTTP/1.1", host: "192.168.50.121:8070"
2021/02/22 09:38:09 [debug] 10296#15732: *65 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:38:09 [debug] 10296#15732: *65 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:38:14 [info] 10296#15732: *65 client prematurely closed connection, so upstream connection is closed too while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070", referrer: "http://192.168.50.121:8070/chrom/pac.pachttp://192.168.50.83:10800/secure/Dashboard.jspa"
2021/02/22 09:38:14 [debug] 10296#15732: *65 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:38:14 [debug] 10296#15732: *65 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:38:21 [info] 10296#15732: *64 client closed connection while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/22 09:38:21 [info] 10296#15732: *62 client prematurely closed connection, so upstream connection is closed too (10054: An existing connection was forcibly closed by the remote host) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:38:21 [debug] 10296#15732: *62 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:38:21 [debug] 10296#15732: *62 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:38:38 [warn] 10296#15732: *59 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:38:38 [warn] 10296#15732: *59 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:38:38 [error] 10296#15732: *59 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:38:38 [error] 10296#15732: *59 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:39:35 [debug] 10296#15732: *67 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:39:35 [debug] 10296#15732: *67 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:39:35 [debug] 10296#15732: *68 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:39:35 [debug] 10296#15732: *68 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:39:35 [debug] 10296#15732: *69 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:39:35 [debug] 10296#15732: *69 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:39:35 [debug] 10296#15732: *70 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:39:35 [debug] 10296#15732: *70 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:39:35 [debug] 10296#15732: *71 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:39:35 [debug] 10296#15732: *71 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:39:38 [debug] 10296#15732: *70 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:39:38 [debug] 10296#15732: *70 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:39:38 [debug] 10296#15732: *67 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:39:38 [debug] 10296#15732: *67 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:39:38 [debug] 10296#15732: *69 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:39:38 [debug] 10296#15732: *69 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:39:38 [debug] 10296#15732: *68 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:39:38 [debug] 10296#15732: *68 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:40:37 [warn] 10296#15732: *59 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:40:37 [warn] 10296#15732: *59 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:40:37 [error] 10296#15732: *59 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:40:37 [error] 10296#15732: *59 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:40:46 [debug] 10296#15732: *75 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:40:46 [debug] 10296#15732: *75 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:37 [warn] 10296#15732: *59 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:41:37 [warn] 10296#15732: *59 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:41:37 [error] 10296#15732: *59 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:41:37 [error] 10296#15732: *59 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:41:42 [debug] 10296#15732: *77 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *77 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *76 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *76 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *78 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *78 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *79 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *79 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *81 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *81 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *80 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *80 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *82 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *82 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *83 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *83 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *84 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:42 [debug] 10296#15732: *84 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:46 [info] 10296#15732: *74 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.159, server: 0.0.0.0:8070
2021/02/22 09:41:47 [debug] 10296#15732: *85 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:47 [debug] 10296#15732: *85 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:41:47 [debug] 10296#15732: *84 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:41:47 [debug] 10296#15732: *84 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:45:07 [warn] 10296#15732: *86 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:45:07 [warn] 10296#15732: *86 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:45:07 [error] 10296#15732: *86 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:45:07 [error] 10296#15732: *86 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:45:09 [info] 10296#15732: *88 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.159, server: 0.0.0.0:8070
2021/02/22 09:45:09 [info] 10296#15732: *89 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.159, server: 0.0.0.0:8070
2021/02/22 09:45:50 [warn] 10296#15732: *90 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070"
2021/02/22 09:45:50 [warn] 10296#15732: *90 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070"
2021/02/22 09:45:50 [error] 10296#15732: *90 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070"
2021/02/22 09:45:50 [error] 10296#15732: *90 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:8082/favicon.ico", host: "192.168.50.121:8070"
2021/02/22 09:45:57 [debug] 10296#15732: *94 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:45:57 [debug] 10296#15732: *94 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:46:07 [warn] 10296#15732: *86 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:46:07 [warn] 10296#15732: *86 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:46:07 [error] 10296#15732: *86 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:46:07 [error] 10296#15732: *86 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:46:07 [info] 10296#15732: *86 client 192.168.50.121 closed keepalive connection
2021/02/22 09:46:32 [debug] 10296#15732: *95 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:46:32 [debug] 10296#15732: *95 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:46:33 [debug] 10296#15732: *96 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:46:33 [debug] 10296#15732: *96 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:46:43 [debug] 10296#15732: *97 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:46:43 [debug] 10296#15732: *97 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:46:43 [debug] 10296#15732: *98 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:46:43 [debug] 10296#15732: *98 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:46:43 [debug] 10296#15732: *99 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:46:43 [debug] 10296#15732: *99 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:46:43 [debug] 10296#15732: *100 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:46:43 [debug] 10296#15732: *100 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:46:44 [debug] 10296#15732: *94 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:46:44 [debug] 10296#15732: *94 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:46:50 [warn] 10296#15732: *90 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070"
2021/02/22 09:46:50 [warn] 10296#15732: *90 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070"
2021/02/22 09:46:50 [error] 10296#15732: *90 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070"
2021/02/22 09:46:50 [error] 10296#15732: *90 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://[::1]:8082/favicon.ico", host: "192.168.50.121:8070"
2021/02/22 09:47:22 [debug] 10296#15732: *94 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:47:22 [debug] 10296#15732: *94 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:47:54 [debug] 10296#15732: *101 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:47:54 [debug] 10296#15732: *101 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:47:54 [debug] 10296#15732: *102 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:47:54 [debug] 10296#15732: *102 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:47:54 [debug] 10296#15732: *103 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:47:54 [debug] 10296#15732: *103 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:47:54 [debug] 10296#15732: *104 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:47:54 [debug] 10296#15732: *104 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:47:55 [debug] 10296#15732: *94 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:47:55 [debug] 10296#15732: *94 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:48:16 [debug] 10296#15732: *105 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:48:16 [debug] 10296#15732: *105 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:49:16 [info] 10296#15732: *106 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/22 09:49:38 [debug] 10296#15732: *107 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:49:38 [debug] 10296#15732: *107 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:49:48 [debug] 10296#15732: *109 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 09:49:48 [debug] 10296#15732: *109 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 09:50:48 [info] 10296#15732: *108 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.159, server: 0.0.0.0:8070
2021/02/22 09:51:10 [warn] 10296#15732: *110 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:51:10 [warn] 10296#15732: *110 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:51:10 [error] 10296#15732: *110 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:51:10 [error] 10296#15732: *110 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:52:10 [warn] 10296#15732: *110 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:52:10 [warn] 10296#15732: *110 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:52:10 [error] 10296#15732: *110 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:52:10 [error] 10296#15732: *110 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:59:37 [warn] 10296#15732: *113 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:59:37 [warn] 10296#15732: *113 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:59:37 [error] 10296#15732: *113 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 09:59:37 [error] 10296#15732: *113 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 10:00:37 [warn] 10296#15732: *113 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 10:00:37 [warn] 10296#15732: *113 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 10:00:37 [error] 10296#15732: *113 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 10:00:37 [error] 10296#15732: *113 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://127.0.0.1:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 10:12:38 [warn] 10296#15732: *116 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 10:12:38 [warn] 10296#15732: *116 upstream server temporarily disabled while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 10:12:38 [error] 10296#15732: *116 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 10:12:38 [error] 10296#15732: *116 upstream timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.50.121, server: localhost, request: "GET /nginx/2/pac HTTP/1.1", upstream: "http://[::1]:8082/nginx/2/pac", host: "192.168.50.121:8070"
2021/02/22 10:12:38 [debug] 10296#15732: *116 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:12:38 [debug] 10296#15732: *116 [lua] log.lua:18: Content-Disposition:inline;filename=pac.pac
2021/02/22 10:12:38 [debug] 10296#15732: *116 [lua] log.lua:20: reverse_value:cap.cap=emanelif;enilni
2021/02/22 10:12:38 [debug] 10296#15732: *116 [lua] log.lua:22: Found = index:8
2021/02/22 10:12:38 [debug] 10296#15732: *116 [lua] log.lua:25: length23
2021/02/22 10:12:38 [debug] 10296#15732: *116 [lua] log.lua:31: Encoded_filename:pac.pac
2021/02/22 10:12:38 [debug] 10296#15732: *116 [lua] log.lua:34: Filename:pac.pac
2021/02/22 10:12:38 [debug] 10296#15732: *116 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:33 [info] 10296#15732: *119 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/22 10:18:33 [info] 10296#15732: *120 client timed out (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while waiting for request, client: 192.168.50.121, server: 0.0.0.0:8070
2021/02/22 10:18:46 [debug] 10296#15732: *121 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *121 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *122 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *122 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *123 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *123 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *124 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *124 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *125 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *125 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *126 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *126 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *127 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *127 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *128 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *128 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *129 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *129 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *130 [lua] log.lua:3: In log_by_lua_file...
2021/02/22 10:18:46 [debug] 10296#15732: *130 [lua] log.lua:38: End log_by_lua_file...
2021/02/22 10:19:07 [notice] 9108#18820: using the "select" event method
2021/02/22 10:19:07 [notice] 9108#18820: using the "select" event method
2021/02/22 10:19:07 [notice] 9108#18820: openresty/1.13.6.2
2021/02/22 10:19:07 [notice] 9108#18820: openresty/1.13.6.2
2021/02/22 10:19:07 [info] 9108#18820: OS: 300000 build:18363, "", suite:300, type:1
2021/02/22 10:19:07 [notice] 9108#18820: start worker processes
2021/02/22 10:19:07 [notice] 9108#18820: start worker processes
2021/02/22 10:19:07 [notice] 9108#18820: start worker process 17400
2021/02/22 10:19:07 [notice] 9108#18820: start worker process 17400
2021/02/22 10:19:07 [notice] 17400#20244: openresty/1.13.6.2
2021/02/22 10:19:07 [notice] 17400#20244: openresty/1.13.6.2
2021/02/22 10:19:07 [info] 17400#20244: OS: 300000 build:18363, "", suite:300, type:1
2021/02/22 10:19:07 [notice] 17400#20244: create thread 14016
2021/02/22 10:19:07 [notice] 17400#20244: create thread 14016
2021/02/22 10:19:07 [notice] 17400#20244: create thread 21436
2021/02/22 10:19:07 [notice] 17400#20244: create thread 21436
2021/02/22 10:19:07 [notice] 17400#20244: create thread 19532
2021/02/22 10:19:07 [notice] 17400#20244: create thread 19532
